src/py42/modules/securitydata.py

```diff
@@ -14,6 +14,15 @@ def __init__(self, security_client, storage_client_factory, microservices_client
         self._client_cache = {}
         self._client_cache_lock = Lock()
 
+    @property
+    def savedsearches(self):
+        """A collection of methods related to retrieving forensic search data.
+
+        Returns:
+            :class: `py42.clients.investigation.SavedSearchClient`
```

Comment:
I think `investigation` should say `securitydata`

Reply:
And `_internal`

Reply:
Copy paste is a sin. Thanks.